doc/RELEASE_WALKTHROUGH.rst

```diff
@@ -154,100 +149,71 @@ Sanity check::
 Tag the release and push the tag. This requires write permission for the numpy
 repository::
 
-    $ git tag -a -s v2.1.0 -m"NumPy 2.1.0 release"
-    $ git push upstream v2.1.0
+    $ git tag -a -s v2.4.0 -m"NumPy 2.4.0 release"
+    $ git push upstream v2.4.0
 
 If you need to delete the tag due to error::
 
-   $ git tag -d v2.1.0
-   $ git push --delete upstream v2.1.0
-
-
-2. Build wheels
----------------
-
-Tagging the build at the beginning of this process will trigger a wheel build
-via cibuildwheel and upload wheels and an sdist to the staging repo. All wheels
-are currently built on GitHub actions and take about 1 1/4 hours to build. 
-
-If you wish to manually trigger a wheel build, you can do so:
-
-- On GitHub actions -> `Wheel builder`_ there is a "Run workflow" button, click
-  on it and choose the tag to build
-
-If some wheel builds fail for unrelated reasons, you can re-run them:
-
-- On GitHub actions select `Wheel builder`_ click on the task that contains
-  the build you want to re-run, it will have the tag as the branch. On the
-  upper right will be a re-run button, hit it and select "re-run failed"
-
-If some wheels fail to upload to anaconda, you can select those builds in the
-`Wheel builder`_ and manually download the build artifact. This is a temporary
-workaround, but sometimes the quickest way to get a release out.
-
-.. _`staging repository`: https://anaconda.org/multibuild-wheels-staging/numpy/files
-.. _`Wheel builder`: https://github.com/numpy/numpy/actions/workflows/wheels.yml
-
-
-3. Download wheels
-------------------
+   $ git tag -d v2.4.0
+   $ git push --delete upstream v2.4.0
 
-When the wheels have all been successfully built and staged, download them from the
-Anaconda staging directory using the ``tools/download-wheels.py`` script::
 
-    $ cd ../numpy
-    $ mkdir -p release/installers
-    $ python3 tools/download-wheels.py 2.1.0
-
-
-4. Generate the README files
-----------------------------
-
-This needs to be done after all installers are downloaded, but before the pavement
-file is updated for continued development::
-
-    $ python write_release 2.1.0
```

Comment:
I'll add this back as ` python tools/write_release 2.4.0` as it translates the release notes into Markdown.